webapp/components/interop.js

```diff
@@ -895,75 +1033,88 @@ class InteropSummary extends PolymerElement {
           display: none;
         }
       </style>
-
-      <div id="summaryContainer">
-        <!-- Chrome/Edge -->
-        <div class="summary-flex-item" tabindex="0">
-          <div class="summary-number">--</div>
-          <template is="dom-if" if="[[stable]]">
-            <div class="summary-browser-name">
-              <figure>
-                <img src="/static/chrome_64x64.png" width="36" alt="Chrome" />
-                <figcaption>Chrome</figcaption>
-              </figure>
-              <figure>
-                <img src="/static/edge_64x64.png" width="36" alt="Edge" />
-                <figcaption>Edge</figcaption>
-              </figure>
-            </div>
-          </template>
-          <template is="dom-if" if="[[!stable]]">
-            <div class="summary-browser-name">
-              <figure>
-                <img src="/static/chrome-dev_64x64.png" width="36" alt="Chrome Dev" />
-                <figcaption>Chrome<br>Dev</figcaption>
-              </figure>
-              <figure>
-                <img src="/static/edge-dev_64x64.png" width="36" alt="Edge Dev" />
-                <figcaption>Edge<br>Dev</figcaption>
-              </figure>
-            </div>
-          </template>
-        </div>
-        <!-- Firefox -->
-        <div class="summary-flex-item" tabindex="0">
-          <div class="summary-number">--</div>
-          <template is="dom-if" if="[[stable]]">
-            <div class="summary-browser-name">
-              <figure>
-                <img src="/static/firefox_64x64.png" width="36" alt="Firefox" />
-                <figcaption>Firefox</figcaption>
-              </figure>
-            </div>
-          </template>
-          <template is="dom-if" if="[[!stable]]">
-            <div class="summary-browser-name">
-              <figure>
-                <img src="/static/firefox-nightly_64x64.png" width="36" alt="Firefox Nightly" />
-                <figcaption>Firefox<br>Nightly</figcaption>
-              </figure>
-            </div>
-          </template>
+      <div class="summary-container">
+        <div id="summaryNumberRow">
+          <!-- Interop -->
+          <div id="interopSummary" class="summary-flex-item" tabindex="0">
+            <h3 class="summary-title">INTEROP</h3>
+            <div class="summary-number score-number">--</div>
+          </div>
+          <!-- Investigations -->
+          <div id="investigationSummary" class="summary-flex-item" tabindex="0">
+            <h3 class="summary-title">INVESTIGATIONS</h3>
+            <div id="investigationNumber" class="summary-number">--</div>
+          </div>
```

Comment:
This section adds the interop and investigation summary bubbles at the top of the page for 2023. The are populated with data similarly to the other summary bubbles. Again, indentations make this look like a larger change.